tests/integration/bootstrap.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Yoast Duplicate Post plugin test file.
+ *
+ * @package Yoast\WP\Duplicate_Post\Tests
+ */
+
+use Yoast\WPTestUtils\WPIntegration;
+
+// Disable xdebug backtrace.
+if ( function_exists( 'xdebug_disable' ) ) {
+	xdebug_disable();
+}
+
+echo 'Welcome to the Duplicate Post Test Suite' . PHP_EOL;
+echo 'Version: 1.0' . PHP_EOL . PHP_EOL;
+
+if ( ! defined( 'YOAST_DUPLICATE_POST_TEST_ROOT_DIR' ) ) {
+	define( 'YOAST_DUPLICATE_POST_TEST_ROOT_DIR', __DIR__ . '/' ); // Includes trailing slash.
+}
+
+/*
+ * Load the plugin(s).
+ */
+require_once dirname( __DIR__, 2 ) . '/vendor/yoast/wp-test-utils/src/WPIntegration/bootstrap-functions.php';
+
+$GLOBALS['wp_tests_options'] = [
+	'active_plugins' => [
+		'duplicate-post/duplicate-post',
+	],
+];
```

Comment:
This shouldn't be needed if the `_manually_load_plugin()` function is defined and hooked in.
You seem to be combining two different bootstrap pattern, resulting in unnecessary bootstrap steps/duplication.

See: https://github.com/Yoast/wp-test-utils#using-the-bootstrap-utilities

Reply:
Removed.